ui/src/components/Tasks/CustomFieldTemplate.jsx

```diff
@@ -30,6 +28,19 @@ export default function CustomFieldTemplate(props) {
   const span = Number(uiSchema['ui:options']?.col) || 6;
   const gridClass = `col-${span}`;
 
+  if (schema.type === 'boolean') {
+    return (
+      <div className={`${gridClass} ${classNames}`.trim()}>
```

Comment:
Maybe move this class name logic to the `gridClass` variable (and rename it if needed)?